src/include/distributed/multi_copy.h

```diff
@@ -0,0 +1,58 @@
+/*-------------------------------------------------------------------------
+ *
+ * multi_copy.h
+ *    Declarations for public functions and variables used in COPY for
+ *    distributed tables.
+ *
+ * Copyright (c) 2016, Citus Data, Inc.
+ *
+ *-------------------------------------------------------------------------
+ */
+
+#ifndef MULTI_COPY_H
+#define MULTI_COPY_H
+
+
+#include "nodes/parsenodes.h"
+
+
+/* config variable managed via guc.c */
+extern int CopyTransactionManager;
+
+
+static const char BinarySignature[11] = "PGCOPY\n\377\r\n\0";
```

Comment:
Declaring a static in a header isn't a good idea; that means it's present in each including file. Afaics there's no need to have in a header at all.